spec/support/suspenders.rb

```diff
@@ -12,9 +12,10 @@ def create_tmp_directory
   def run_suspenders(arguments = nil)
     Dir.chdir(tmp_path) do
       Bundler.with_clean_env do
-        ENV['TESTING'] = '1'
-
-        %x(#{suspenders_bin} #{APP_NAME} #{arguments})
+        `
```

Comment:
Okay.

Something I just noticed -- is `add_fakes_to_path` not working to set the `PATH`? Is this why you had to add `export PATH` here?